app/src/main/res/values/strings.xml

```diff
@@ -22,5 +26,6 @@
 
     <!-- Temp -->
     <string name="text_temp_connect">Connect UI</string>
```

Comment:
You can delete the two strings "text_temp_connect" and "text_temp_launch" if you're not using them anymore